podium-gateway/src/main/resources/config/application-dev.yml

```diff
@@ -72,12 +75,11 @@ spring:
         username:
         password:
     messages:
-        cache-seconds: 1
+        cache-duration: 1
     thymeleaf:
         cache: false
-
-liquibase:
-    contexts: dev
+    liquibase:
```

Comment:
Are you sure?

Reply:
Then also in application-prod.yml

Reply:
Yes. You can see `spring.liquibase.contexts` option in [Liquibase docs](https://docs.liquibase.com/tools-integrations/springboot/springboot.html).

Reply:
For both podium-gateway and podium-uaa